arrays/dmytro/C#_soft-187/Task_9/Task_9/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Task_9
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] array = { -3, 5, -6, 12, -22, 27 };
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (array[i] > 0)
+                {
+                    ++array[i];
+                    Console.WriteLine(array[i]);
+                }
+                if (array[i] < 0)
```

Comment:
Why do we need use two if conditions?
Much better to use if else construction. Rework it.